api/src/serge/models/user.py

```diff
@@ -0,0 +1,41 @@
+from sqlalchemy import (Boolean, Column, ForeignKey, Integer, String,
+                        Uuid)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Uuid, primary_key=True)
+    username = Column(String, unique=True, index=True)
+    email = Column(String)
+    full_name = Column(String)
+    theme_light = Column(Boolean)
+    default_prompt = Column(String)
+    is_active = Column(Boolean, default=True)
+
+    auth = relationship("UserAuth", back_populates="user", lazy="joined")
+    chats = relationship("Chat", back_populates="user", lazy="joined")
+
```

Comment:
@Jonpro03 Thoughts on this?

Reply:
> @Jonpro03 Thoughts on this?

Email isn't (currently) required to create an account, so putting unique on the column fails when the second user creates an account with no email (from testing), so I removed it.

Email in general doesn't really serve a purpose atm, because there's no self-service password reset. So mostly a placeholder for the future when I do 3rd party auth.